lemur/plugins/lemur_entrust/plugin.py

```diff
@@ -293,12 +293,33 @@ def revoke_certificate(self, certificate, reason):
 
     @retry(stop_max_attempt_number=3, wait_fixed=1000)
     def deactivate_certificate(self, certificate):
-        """Deactivates an Entrust certificate."""
+        """Deactivates an Entrust certificate, as long as it is still active, and not already deactivcated. """
+        log_data = {
+            "function": f"{__name__}.{sys._getframe().f_code.co_name}",
+            "external_id": f"{certificate.external_id}"
+        }
+
+        # Let's first check the status of the certificate
         base_url = current_app.config.get("ENTRUST_URL")
-        deactivate_url = f"{base_url}/certificates/{certificate.external_id}/deactivations"
-        response = self.session.post(deactivate_url)
-        metrics.send("entrust_deactivate_certificate", "counter", 1)
-        return handle_response(response)
+        deactivate_url = f"{base_url}/certificates/{certificate.external_id}"
```

Comment:
Is this more like a `status_url`?

Reply:
oh good catch, I think we can designate it as such. It fetches the entire certificate at this point, to avoid failures on deactivation.